src/main/java/ListOfItems.java

```diff
@@ -0,0 +1,353 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
```

Comment:
Agreed, it would probably be better if the `java.util.ArrayList` is separated in a new line.